app/views/needs/index.html.erb

```diff
@@ -1,11 +1,16 @@
 <p id="notice"><%= notice %></p>
 
 <div class="page-header">
-  <h1 class="page-title">Needs <span class="page-title__count">(<%= @needs.total_count %>)</span></h1>
+  <h1 class="page-title">Needs</h1>
 </div>
 
-     
-
+<% if @needs.total_count > 0 %>
+  <div class="filtered-notice notice">
+    Showing <%= pluralize(@needs.total_count, 'result') %>
+    for the current filters.
```

Comment:
Wouldn't this show even if no filters are set? And the "Clear filters" link would always be there too, right?